index.bs

```diff
@@ -10796,11 +10795,30 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 				[=Computing a block of audio|compute a block of audio=], and
 				[=Making a buffer available for reading|make it available for reading=].
 
-			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, invoke associated
-				{{AudioWorkletProcessor}}'s {{AudioWorkletProcessor/process()}} method
-				to [=Computing a block of audio|compute a block of audio=] with
-				the argument of [=input buffer=], output buffer and
-				[=input AudioParam buffer=].
+			4. If this {{AudioNode}} is an {{AudioWorkletNode}}, execute these steps:
+
+			  1. If <a>callable process</a> is `false`, abort the following sub steps.
```

Comment:
I think you need to indent this some more.  I see that this step is labeled 5.

Also, when the following steps are aborted, where do we go from here?  What's the next step in the algorithm?